bundle/regal/lsp/completion/providers/packagename/packagename.rego

```diff
@@ -0,0 +1,51 @@
+package regal.lsp.completion.providers.packagename
+
+import rego.v1
+
+import data.regal.lsp.completion.kind
+import data.regal.lsp.completion.location
+
+items contains item if {
+	position := location.to_position(input.regal.context.location)
+	line := input.regal.file.lines[position.line]
+
+	invoke_suggestion(line, position)
+
+	ps := input.regal.context.path_separator
+
+	abs_dir := base(input.regal.file.name)
+	rel_dir := trim_prefix(abs_dir, input.regal.context.workspace_root)
+	fix_dir := replace(replace(trim_prefix(rel_dir, ps), ".", "_"), ps, ".")
```

Comment:
oh my 🙈

Reply:
heh, yeah, not sure how that could be made more clear... perhaps an extra intermediate assignment or something